### DISCUSSION TITLE
DWAA/DWAB compression amount

### DISCUSSION THREAD

**Discussion by russellkoonce** [ImageMagick/ImageMagick#3931] [Digital Image Processing]
Are there quality settings available for DWA compression? 
Typically the default compression percentage is 45% but I don't see a way to specify in the command line options.

**Comment by snibgo** [comment]
I don't think there is an option to directly control the amount of compression. For some compression methods, "-quality" affects the filesize, but this doesn't seem to be the case for "-compress DWAA" or "-compress DWAB".

**Reply by russellkoonce** [reply]
Thank you. I saw the same results when I tested it.
